Marlin/src/pins/stm32f4/pins_LERDGE_X.h

```diff
@@ -162,15 +165,24 @@
 
 #define LCD_RESET_PIN                       PD6
 #define LCD_BACKLIGHT_PIN                   PD3
-#define FSMC_CS_PIN                         PD4
-#define FSMC_RS_PIN                         PD11
-#define TOUCH_CS                            PB6
+#define TFT_CS_PIN                          PD4
+#define TFT_RS_PIN                          PD11
```

Comment:
Marlin doesn't know these pin names, so I assume the previous pins names were correct.